### ISSUE TITLE
"Hidden" links

### ISSUE THREAD

**Issue by agzam** [org-roam/org-roam#1657]
When there are too many links in the DB, it becomes difficult to navigate through them. Could we figure out a way to have links that:

- can be stored `(org-store-link)` 
- can be inserted `(org-insert-link)`
- would appear in the backlinks [`*org-roam*`] buffer

But would be hidden (perhaps by default) in the list of links - `(org-roam-node-find)` and similar functions won't show them?
_Does this sound like a really dumb idea?_

**Comment by jethrokuan** [comment]
You can do this by:

1. Creating your own link type (like `my-id:foo`)
2. Using that link type to insert nodes
3. Create a widget for your custom link type

None of the functions concern themselves with links though, so I'm not sure what your ask is here.

**Comment by agzam** [comment]
I'm not sure what "Create a widget for your custom link type" means. 

I'm sorry, maybe I have not formulated the idea properly. It looks like what I'm asking is akin to `:ROAM_EXCLUDE: t` (I just learned about its existence), with the difference that it only exclude it from `org-roam-node-find`, but would keep it in the backlinks buffer (when navigated to a node that has a link to it).

I guess this feature of the "partial exclude" is somewhat "esoteric" and I think I know how to implement it in my own config. 

If someone else asks for this again, perhaps it could be done, with modifying `:ROAM_EXCLUDE:` so it could take other values,  not only `t`. But of course you have bigger fish to fry. Closing this for now.
